CONTRIBUTING.md

```diff
@@ -11,6 +11,11 @@ Instead, modify the sources in the ```www``` folder and compile a new version of
 
 ```node.js``` and ```npm``` are needed for building this project. ```npm``` comes bundled with the ```node.js``` installer. You can download the ```node.js``` installer here: https://nodejs.org/download/. 
 
+To run Android tests, make sure you have ```sdk\tools``` and  ```sdk\platform-tools``` in your PATH.
+
+If you would like the tests to start the emulators for you, configure an Android emulator using the AVD and name it ```emulator```.
```

Comment:
jk, just talked with Geoffrey, gonna leave the enumeration code in because the union rules (android + ios-ui + ios-wk = test, ios-ui + ios-wk = ios, etc) makes supplying what tests to run kinda impractical, but emulator can be passed by command line